### PR TITLE
Add start and watch to Makefile

### PR DESCRIPTION
**Pull request by frankieroberto**
This is just to help remind us how to run the nanoc server locally (which I had to look up).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,3 +9,6 @@ npm_install:
 
 bundle_install:
 	bundle install
+
+start:
+	nanoc view
```
Comment (peteryates) [inline Makefile]: Good idea. I think `bundle exec` is going to make things work more predictably here too.

```suggestion
start:
	bundle exec nanoc view
```

Comment (peteryates) [inline Makefile]: Additionally there's `nanoc live` which does live reloading when things change. I used `view` when doing template work as the live reloader went a bit nuts refreshing for 900 pages, but in everyday work it's probably what people want.

```suggestion
start:
	nanoc view
	
watch:
  bundle exec nanoc live
```

File: Makefile [hunk 2]
```diff
@@ -10,5 +10,11 @@ npm_install:
 bundle_install:
 	bundle install
 
+start:
+	bundle exec nanoc view
+
+watch:
+  	bundle exec nanoc live
```
Comment (peteryates) [inline Makefile]: Make won't like spaces before tabs. I love Make but it's awfully fussy!

```suggestion
	bundle exec nanoc live
```

Comment (frankieroberto) [reply Makefile]: Ugh sorry - I was editing it via the GitHub web UI so didn't spot the spaces...

**Pull request merged**